bftengine/src/bftengine/messages/ClientRequestMsg.cpp

```diff
@@ -81,7 +82,7 @@ void ClientRequestMsg::setParams(NodeIdType sender, ReqId reqSeqNum, uint32_t re
   setParams(reqSeqNum, requestLength, flags);
 }
 
-std::string ClientRequestMsg::getCid() {
+const std::string ClientRequestMsg::getCid() const {
```

Comment:
[Minor]: Not sure if we want to return a `const` string here.

Reply:
Isn't it a good idea such that no one will be able to change the return value?